YOLO.xcodeproj/project.pbxproj

```diff
@@ -351,7 +331,7 @@
 				ASSETCATALOG_COMPILER_INCLUDE_ALL_APPICON_ASSETS = NO;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 0;
-				DEVELOPMENT_TEAM = 3MR4P6CL3X;
+				DEVELOPMENT_TEAM = "";
```

Comment:
@glenn-jocher, What do you think about this? Should we leave our code or should we make a blank one?